kitsune/questions/views.py

```diff
@@ -1210,16 +1196,17 @@ def edit_answer(request, question_id, answer_id):
             request, "questions/edit_answer.html", {"form": form, "answer": answer}
         )
 
-    form = AnswerForm(request.POST)
+    form = AnswerForm(request.POST, **{'user': request.user})
```

Comment:
Is there a reason for unpacking the dict instead of using keyword args? I'm genuinely curious